apollo-client/src/test/java/com/ctrip/framework/apollo/internals/ConfigServiceLocatorTest.java

```diff
@@ -27,15 +28,15 @@
 
   @After
   public void tearDown() throws Exception {
-    System.clearProperty("apollo.configService");
+    System.clearProperty(ApolloClientSystemConsts.APOLLO_CONFIG_SERVICE);
   }
 
   @Test
   public void testGetConfigServicesWithSystemProperty() throws Exception {
     String someConfigServiceUrl = " someConfigServiceUrl ";
     String anotherConfigServiceUrl = " anotherConfigServiceUrl ";
 
-    System.setProperty("apollo.configService", someConfigServiceUrl + "," + anotherConfigServiceUrl);
+    System.setProperty(ApolloClientSystemConsts.APOLLO_CONFIG_SERVICE, someConfigServiceUrl + "," + anotherConfigServiceUrl);
```

Comment:
How about create test to ensure the compability with deprecated key `apollo.configService`? Just a simple copy, add suffix `Deprecated` on method, change `APOLLO_CONFIG_SERVICE` to `DEPRECATED_APOLLO_CONFIG_SERVICE`. 

Or left it to other pull request.

like follow:
```java

  @After
  public void tearDown() throws Exception {
    System.clearProperty(ApolloClientSystemConsts.DEPRECATED_APOLLO_CONFIG_SERVICE);
    System.clearProperty(ApolloClientSystemConsts.APOLLO_CONFIG_SERVICE);
  }

  @Test
  public void testGetConfigServicesWithSystemProperty() throws Exception {
    String someConfigServiceUrl = " someConfigServiceUrl ";
    String anotherConfigServiceUrl = " anotherConfigServiceUrl ";

    System.setProperty(ApolloClientSystemConsts.APOLLO_CONFIG_SERVICE, someConfigServiceUrl + "," + anotherConfigServiceUrl);

    ConfigServiceLocator configServiceLocator = new ConfigServiceLocator();

    List<ServiceDTO> result = configServiceLocator.getConfigServices();

    assertEquals(2, result.size());

    assertEquals(someConfigServiceUrl.trim(), result.get(0).getHomepageUrl());
    assertEquals(anotherConfigServiceUrl.trim(), result.get(1).getHomepageUrl());
  }

  @Test
  public void testGetConfigServicesWithSystemPropertyDeprecated() throws Exception {
    String someConfigServiceUrl = " someConfigServiceUrl ";
    String anotherConfigServiceUrl = " anotherConfigServiceUrl ";

    System.setProperty(ApolloClientSystemConsts.DEPRECATED_APOLLO_CONFIG_SERVICE, someConfigServiceUrl + "," + anotherConfigServiceUrl);

    ConfigServiceLocator configServiceLocator = new ConfigServiceLocator();

    List<ServiceDTO> result = configServiceLocator.getConfigServices();

    assertEquals(2, result.size());

    assertEquals(someConfigServiceUrl.trim(), result.get(0).getHomepageUrl());
    assertEquals(anotherConfigServiceUrl.trim(), result.get(1).getHomepageUrl());
  }
```

Reply:
add the compatible unit test